backend/src/test/java/ufrn/imd/boraPagar/user/UserTests.java

```diff
@@ -123,4 +133,22 @@ public void shouldPageOneHasLengthZero() {
 
         Assert.assertEquals(0, userPage.getContent().size());
     }
+
+    @Test
+    public void shouldPageZeroHasFirstTheObjectUserUsingOrderByName() {
+        Pageable pageable = PageRequest.of(0, 5);
+        Page<UserModel> userPage = repository.findAllByNameOrderByNameAsc(pageable);
+
+        Assert.assertEquals(user, userPage.getContent().get(0));
+        Assert.assertEquals(user2, userPage.getContent().get(1));
+    }
+
+    @Test
+    public void shouldPageZeroHasFirstTheObjectUserUsingOrderByUsername() {
```

Comment:
Acho que o nome do teste deveria ser:
```suggestion
    public void shouldPageZeroHasUserAsFirstObjectUsingOrderByUsername() {
```